vms/administrator/templates/administrator/pdf.html

```diff
@@ -0,0 +1,47 @@
+<html>
+<body>
+{{report.date_submitted}}<br>
+AnitaB.org<br>
+1501 Page Mill Road<br>
+MS 1105<br>
+Palo Alto, CA 94304<br>
+<br>
+To Whom It May Concern:<br> 
+<br>
+Re: {{report.volunteer.first_name}} {{report.volunteer.last_name}}<br>
+<br>
+This letter is to confirm that {{report.volunteer.first_name}} {{report.volunteer.last_name}} completed {{report.total_hrs}} of unpaid volunteer work at AnitaB.org as a {% for report in report_list %}{{report.job_name}}{% endfor %}.
+
+<hr />
+<div class="well">
+   <table class="table table-striped table-hover">
+            <td>Event Name</td>
+            <td>Job Name</td>
+            <td>Date</td>
+            <td>Logged Start Time</td>
+            <td>Logged End Time</td>
+            <td>Duration</td>
+         {% for report in report_list %}
+         <tr>
+            <td>{{report.event_name}}</td>
+            <td>{{report.job_name}}</td>
+            <td>{{report.date}}</td>
+            <td>{{report.logged_start_time}}</td>
+            <td>{{report.logged_end_time}}</td>
+            <td>{{report.duration}}</td>
+         </tr>
+         {% endfor %}
+   </table>
+</div>
+<hr />
+
+<br>
+If you require further confirmation of the report generated above, please contact {{admin.first_name}} {{admin.last_name}} at {{admin.email}}.
```

Comment:
@anjali-dhanuka  @abhishekarya286  According to me there should be a link to admin's profile rather than just their email id?

Reply:
Yes, that will make more sense.

Reply:
@kamsuri Can the volunteers contact the Admin through the profile? If not, I don't think there's a point to refer them to the profile.

Reply:
There doesn't exist any admin profile viewable by volunteers for now.